src/components/v5/common/ActionSidebar/hooks/useGetColonyAction.ts

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { failedLoadingDuration as pollingTimeout } from '~frame/LoadingTemplate';
+import {
+  ColonyActionType,
+  useGetColonyActionQuery,
+  useGetMotionStateQuery,
+} from '~gql';
+import { isTransactionFormat } from '~utils/web3';
+import { useColonyContext } from '~hooks';
+import { useUserTokenBalanceContext } from '~context';
+
+export type RefetchMotionState = ReturnType<
+  typeof useGetMotionStateQuery
+>['refetch'];
+
+export type RefetchAction = ReturnType<
+  typeof useGetColonyActionQuery
+>['refetch'];
```

Comment:
Can you move this types to separate file?

Reply:
I will clean up this file later, when we will add a logic to each step form